docs/reST/ref/font.rst

```diff
@@ -472,6 +472,23 @@ solves no longer exists, it will likely be removed in the future.
 
       .. ## Font.get_descent ##
 
+   .. method:: set_script
+
+      | :sl:`set the script code for text shaping`
+      | :sg:`set_script(str) -> None`
+
+      **Experimental:** feature still in development available for testing and feedback. It may change.
+      `Please leave feedback with authors <https://github.com/pygame/pygame/pull/3330>`_
+
+      Sets the script used by harfbuzz text shaping, taking a 4 character
```

Comment:
Docs should probably mention that this function is only supported on SDL_ttf 2.20 or higher

Reply:
Done